packages/fxa-content-server/app/tests/spec/views/settings/delete_account.js

```diff
@@ -367,8 +363,8 @@ describe('views/settings/delete_account', function() {
 
       describe('_uniqueBrowserNames', () => {
         beforeEach(() => {
-          activeSubscriptions = [];
-          attachedClients = [
+          activeSubscriptions.splice(0);
+          sinon.stub(attachedClients, 'toJSON').returns([
```

Comment:
`activeSubscriptions` and `attachedClients` have already been passed into the mocked view by this point, so reassigning them here is just playing with local variables. Modifying them in-place instead sets the test case up as expected.

(and same thing below)